### ISSUE TITLE
Resource helm_release doesn't support import

### ISSUE THREAD

**Issue by bluemalkin** [hashicorp/terraform-provider-helm#113]
Despite the docs at https://github.com/mcuadros/terraform-provider-helm/blob/master/docs/release.md#import saying you can import, I get an error message:

```
terraform import module.sqlproxy-pg.helm_release.this pg-sqlproxy
module.sqlproxy-pg.helm_release.this: Importing from ID "pg-sqlproxy"...

Error: module.sqlproxy-pg.helm_release.this (import id: pg-sqlproxy): import module.sqlproxy-pg.helm_release.this (id: pg-sqlproxy): resource helm_release doesn't support import
```

**Comment by colinhoglund** [comment]
From what I can tell, helm does not provide a way to retrieve the source repository that installed a release, and the `repository` schema is currently set to `ForceNew: true` (https://github.com/mcuadros/terraform-provider-helm/blob/master/helm/resource_release.go#L48). So importing currently does not really seem possible without some changes to the terraform schema, at a minimum, or by manually editing the state file.

**Comment by colinhoglund** [comment]
Should these issues be moved over to https://github.com/terraform-providers/terraform-provider-helm?

**Comment by rporres** [comment]
Already moved, @colinhoglund ;-)

**Comment by sebglon** [comment]
We wait for a release...

**Comment by iptizer** [comment]
How is the status regarding the import of resources with the provider 1.0.0 release?

**Comment by houstonj1** [comment]
This would be an awesome feature with the 1.0.0 release!

**Comment by legal90** [comment]
Hi guys! You can track the progress here: https://github.com/terraform-providers/terraform-provider-helm/pull/394

**Comment by mcuadros** [comment]
Fixed by #394
